app/src/main/java/com/lastaoutdoor/lasta/ui/screen/favorites/FavoritesScreen.kt

```diff
@@ -45,8 +54,9 @@ fun FavoritesScreen(
     HorizontalDivider()
     if (isLoading) {
       LoadingAnim(width = 35, tag = "LoadingBarFavorites")
-    } else if (favorites.isEmpty()) {
-      /*TODO*/
+    } else if (favorites.isEmpty() || favorites[0].isEmpty()) {
+      // the second condition is to avoid being baited by a potential empty favorite activity

```

Comment:
I don't really see when this will be relevant, but if you have a good reason.. (we can't have empty activities)

Reply:
Yes in fact it can happen that the `favorites` list is not empty but contains only an empty activity. 
Therefore the "no favorites" message is not displayed but neither is the activity since it is empty. Thus this check is necessary

Reply:
I see !